python/cuml/svm/svm_base.pyx

```diff
@@ -453,7 +453,7 @@ class SVMBase(Base):
             else:
                 self._unique_labels = None
 
-    def predict(self, X, predict_class):
+    def predict(self, X, predict_class, convert_dtype=False):
```

Comment:
By keeping the default False on the base class we keep the clean separation for the regression case.

Reply:
Not a big deal, but since we want the norm to be True, wouldn't it be better for it to be opt_out, i.e. the class that inherits and doesn't want the behavior has to explicitly do convert_dtype=False

Reply:
As usual, that makes more sense :)

Reply:
Resolved by https://github.com/rapidsai/cuml/pull/2723/commits/ddb55480362ecb66120d981621b5f0b7ceea19a5